CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Here's all notable changes and commits to both the configuration repo and the ba
 Many thanks to all those who have submitted issues and pull requests to make this firmware better!
 ## Config repo
 
+2/1/2024 - Add targets to makefile to allow for cleaning firmware and docker image separately while maintaining functionality of make clean to run both
```

Comment:
Would you be able to link the PR in these changelog lines like the other changelog lines?

Reply:
Will do. Would you want me to squash all changes into a single commit? I'm considering squashing and changing the commit message to better reflect all makefile changes as it is a bit more than just allowing left side build only.

Reply:
Squashing into one would be appreciated